biz.aQute.bndlib/src/aQute/bnd/osgi/FileResource.java

```diff
@@ -51,7 +52,7 @@ public String toString() {
 	}
 
 	public void write(OutputStream out) throws Exception {
-		copy(this, out);
+		Files.copy(file.toPath(), out);
```

Comment:
Why this change? It was using the method below which can be overridden. I don't see the point of this change. I don't think this change should be made.